src/pyscipopt/scip.pxd

```diff
@@ -348,7 +351,8 @@ cdef extern from "scip/scip.h":
         pass
 
     ctypedef struct SCIP_BRANCHRULE:
-        pass
+        SCIP_RETCODE (*branchexeclp)(SCIP* scip, SCIP_BRANCHRULE* branchrule, SCIP_Bool allowaddcons, SCIP_RESULT* result)
+        const char* SCIPbranchruleGetName(SCIP_BRANCHRULE* branchrule)
```

Comment:
This should appear after SCIPbranchruleGetData below.

Reply:
Yes, indeed.